security/file_integrity_operator/file-integrity-operator-configuring.adoc

```diff
@@ -0,0 +1,107 @@
+[id="configuring-file-integrity-operator"]
+= Configuring the Custom File Integrity Operator
+include::modules/common-attributes.adoc[]
+:context: file-integrity-operator
+toc::[]
+
+
+== Understanding the FileIntegrity object
+Let’s explore the . As usual with any Kubernetes CRs, you can call:
+$ oc explain fileintegrity
+And then look at the individual attributes:
+$ oc explain fileintegrity.spec
+$ oc explain fileintegrity.spec.config
```

Comment:
needs to be code in markdown

Reply:
@JAORMX Thanks! I am still getting the first draft together and will be in touch soon for a review.